apps/server/src/modules/task/service/task.service.ts

```diff
@@ -70,6 +72,16 @@ export class TaskService {
 			const course = await this.courseRepo.findById(params.courseId);
 			this.authorizationService.checkPermission(user, course, PermissionContextBuilder.write([]));
 			taskParams.course = course;
+
+			if (params.usersIds) {
+				const courseUsers = course.getStudentIds();
+				const isAllUsersInCourse = params.usersIds.every((id) => courseUsers.includes(id));
+				if (!isAllUsersInCourse) {
+					throw new BadRequestException('Users do not belong to course');
```

Comment:
does `ForbiddenException` make sense here? at least it came to my mind

Reply:
yes I think you are right. Forbidden is more appropiate, as there is nothing wrong with the request format